### DISCUSSION TITLE
I would like Aggregate Verifiers to be issued

### DISCUSSION THREAD

**Discussion by tech-yda** [Web3Auth/Web3Auth#1073] [Help]
Hello.
In our app we need to handle firebaseAuth jwt authentication and passwordless authentication with the same account.
I think i need Single ID Verifier, but the document says that you need to write it in github discussion to request issuance.
Would you please?

Thank you.

**Comment by Saviour1001** [comment]
Hey there, we launched this feature this week where you can create the Aggregate Verifier from the developer dashboard itself.  
<img width="400" alt="image" src="https://user-images.githubusercontent.com/71517788/207717481-6d6f895f-1d85-4ceb-9385-af00eb1908b8.png">

Continue building !!

**Reply by tech-yda** [reply]
Thank you for your reply!

I checked this feature.
Please let me ask you an additional question.
Is it possible to do passwordless authentication to an account authenticated by firebaseAuth without using auth0?

I want to log in to OpenSea without going through WalletConnect with a currency wallet created via web3auth by a user who registered with web3auth with firebaseAuth authentication.

**Reply by Saviour1001** [reply]
Yes, passwordless via FirebaseAuth is possible, you can set it up using Firebase Dashboard. 

![image](https://user-images.githubusercontent.com/71517788/207844811-ccb585f9-1fa9-4cda-a26b-28ed11ff2bea.png)

Open Sea is app scoped and thus not sure if the pipeline will work as you wrote.

**Reply by yukitch** [reply]
@Saviour1001 Sorry to bother you. We would like to use passwordless login for a custom authentication user (firebase auth jwt token), because a custom authentication user cannot login OpenSea (I know we can use walletConnect via torus ui but torus ui is not best for us) and they also cannot login into torus wallet with custom authentication login.
So we would like allow the same user who logged in with custom authentication to also log in with passwordless login.

I also tried aggregate verifier, but I can not find passwordless login as sub verifier. Is this for social login provider only?
Do you have any other solution?

I look forward to hearing from you.
